lib/dams/dams_object.rb

```diff
@@ -597,6 +597,22 @@ def to_solr (solr_doc = {})
 	      insertFacets solr_doc, "subject_topic", load_series(component.series)
 	      insertFacets solr_doc, "subject_topic", load_cruises(component.cruise)
 	      	      	
+          # uncomment for subject anatomy
+          #insertFacets solr_doc, "subject_anatomy", load_anatomys(component.anatomy)
+          #insertFacets solr_doc, "component_#{cid}_subject_anatomy", load_anatomys(component.anatomy)
+          insertFacets solr_doc, "subject_common_name", load_commonNames(component.commonName)
+          insertFacets solr_doc, "component_#{cid}_subject_common_name", load_commonNames(component.commonName)
```

Comment:
well, i'm unclear what those method calls are actually doing. because they're running the insertFacets method and specifying a facet name that doesn't exist, or at least isn't created in the catalog controller. So, I'd like to think we have no use for them and could safely remove them. Perhaps you could verify by adding a test that has a component-level common name or scientific name, and then assert that a search that matches the record shows the facet on the results view? Or something like that.